packages/front/svelte/components/Router/Router.controller.js

```diff
@@ -13,17 +13,18 @@ export default class RouterController {
     }
 
     loadRoute(path, searchQuery) {
-        const current = RouterService.getRoute(this.routes, path);
-        if (!current.disableAuth) {
+        const route = RouterService.getRoute(this.routes, path);
+        if (!route) return RouterService.goToUrl("/404");
```

Comment:
Chez pas si c'est légal ça
je me demande pas si on doit pas afficher une 404 mais garder la même url

Reply:
C'est pas faux. En plus au début j'avais fait `if (!route) route = RouterService.getRoute(this.routes, "/404")`

Tu trouves ça plus légitime ?

Reply:
J'ai rien compris :D ! On en reparle lundi :p

Reply:
> C'est pas faux. En plus au début j'avais fait `if (!route) route = RouterService.getRoute(this.routes, "/404")`
> 
> Tu trouves ça plus légitime ?

Après test en local je pense que tu as mis en commentaire me paraît mieux. `goToUrl` refresh toute la page, c'est pas très fluide. Mais ça marche pas non plus très bien pour le reste de la fonction que le "vrai" path ne coïncide pas avec la route donc à fouiller

Reply:
du coup j'ai appliqué mon commentaire et on affiche la page 404 en gardant l'url de base dans la barre de navigation